exercises/rotationa-cipher

```diff
@@ -0,0 +1,2 @@
+def rotational_cipher():
```

Comment:
I assume this file was exceedingly created (at least it's not needed) and should be removed.